src/libs/blueprint/conduit_blueprint_mesh_utils.hpp

```diff
@@ -984,14 +1165,16 @@ namespace adjset
 
      @param doms A node containing the domains. There must be multiple domains.
      @param adjsetName The name of the adjset in all domains. It must exist.
+     @param options An options node. Include "tolerance" to set the point tolerance.
      @param[out] info A node that contains any errors.
 
      @return True if the adjsets in all domains contained no errors; False if
              there were errors.
      */
-    bool CONDUIT_BLUEPRINT_API validate(const Node &doms,
+    bool CONDUIT_BLUEPRINT_API validate(const conduit::Node &doms,
                                         const std::string &adjsetName,
-                                        Node &info);
+                                        const conduit::Node &options,
```

Comment:
Should the options node be an optional argument?

Reply:
I changed the `options` argument so it comes last and is an optional argument, defaulted to `conduit::Node()`. I updated places in Conduit where the function is called. I did this for adjset `validate` and `compare_pointwise` functions.